lib/generators/react_on_rails/templates/base/base/config/webpack/test.js.tt

```diff
@@ -0,0 +1,13 @@
+<%= add_documentation_reference(config[:message], "// https://github.com/shakacode/react_on_rails_tutorial_with_ssr_and_hmr_fast_refresh/blob/master/config/webpack/development.js") %>
+
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
+
+const { devServer, inliningCss } = require('@rails/webpacker');
+
+const webpackConfig = require('./webpackConfig');
+
+const developmentEnvOnly = (clientWebpackConfig, _serverWebpackConfig) => {
+  // place any code here that is for test only
+};
+
+module.exports = webpackConfig(developmentEnvOnly);
```

Comment:
@Judahmeek this file is for NODE_ENV=test for Jest testing, and not Rails, FYI.

So I don't see the value of this file. It could just be the default.

Reply:
https://rossta.net/blog/why-doesnt-webpacker-use-my-test-config-when-i-run-rails-tests.html

Reply:
My bad. I vaguely remembered that discussion, but I couldn't remember the exact reason.

I'll remove it.